src/main/java/module-info.java

```diff
@@ -150,4 +150,5 @@
     requires dd.plist;
     requires mslinks;
     requires org.apache.httpcomponents.core5.httpcore5;
+    requires commons.beanutils;
```

Comment:
Why was this added?

Reply:
I am not sure - I think it was auto-imported by IntelliJ, which is how I also ended up with so many .* imports. I couldn't find any other mentions of an import from it, so I can remove it and see how the build goes.

Reply:
This looks like an observable import from the wrong package